homeassistant/components/netgear/sensor.py

```diff
@@ -374,4 +377,12 @@ def async_update_device(self) -> None:
         """Update the Netgear device."""
         if self.coordinator.data is not None:
             data = self.coordinator.data.get(self.entity_description.key)
-            self._value = self.entity_description.value(data)
+            if data is not None:
```

Comment:
I'd invert the check, if true, reset the value, do the debug log and return. Then outdent below and have the normal flow the most outdented.

Reply:
done, see above.